packages/Ludown/lib/ludown.js

```diff
@@ -7,27 +7,36 @@ const program = require('commander');
 const chalk = require('chalk');
 const pjson = require('../package.json');
 const utils = require('./utils');
-program
-    .option('--prefix', 'Add [ludown] prefix to all messages')
-    .on('option:prefix', () => process.env.PREFIX = 'prefix');
-program.Command.prototype.unknownOption = function () {
-    process.stderr.write(chalk.default.redBright(`\n  Unknown arguments: ${process.argv.slice(2).join(' ')}\n`));
-    program.help();
-};
+const semver = require('semver');
+const getLatestVersion = require('latest-version');
+getLatestVersion(pjson.name).then((latest) => {
+    if (semver.gt(latest, pjson.version)) {
+        process.stderr.write(chalk.default.yellowBright(`\nNew version ${latest} is available to install.\n\n`));
```

Comment:
Consider replacing this with these lines (reviewed text with design). 

        process.stderr.write(chalk.default.white(`\n     Update available`));
        process.stderr.write(chalk.default.gray(` ${pjson.version}`));
        process.stderr.write(chalk.default.white(` -> `));
        process.stderr.write(chalk.default.greenBright(`${latest}\n`));
        process.stderr.write(chalk.default.white(`       Run `));
        process.stderr.write(chalk.default.blueBright(`npm i -g ${pjson.name}`));
        process.stderr.write(chalk.default.white(` to update.\n\n`));